vmb/matrimony/models/profiles.py

```diff
@@ -580,7 +581,7 @@ def send_batch_matches_email(self):
 
     def get_batch_matches_email_body(self):
         return loader.get_template("matrimony/emails/matches.html").render(
-            {"matches": self.matching_profiles}
+            {"profile": self}
```

Comment:
Why expose `profile` here? I don't see usage of `profile` but only `matches` in the template.